instrumentation/opentelemetry-instrumentation-flask/src/opentelemetry/instrumentation/flask/__init__.py

```diff
@@ -309,7 +309,16 @@ def response_hook(span: Span, status: str, response_headers: List):
 
 flask_version = version("flask")
 
-if package_version.parse(flask_version) >= package_version.parse("2.2.0"):
+if package_version.parse(flask_version) >= package_version.parse("3.1.0"):
```

Comment:
Isn't this the same of >=2.2.0 with just the exceptions catched? If so I guess we can just add the exceptions catching to that version